README.md

```diff
@@ -1,28 +1,21 @@
 # Watson Developer Cloud Swift SDK: Chat App Example
 
-This repository contains an example application to demonstrate how the Swift-based [Watson Developer Cloud Swift SDK](https://github.com/watson-developer-cloud/ios-sdk) can be consumed from an Objective-C application.
+This repository contains an example application to demonstrate how the Swift-based [Watson Developer Cloud Swift SDK](https://github.com/watson-developer-cloud/swift-sdk) can be consumed from an Objective-C application.
 
 This example modifies the [JSQMessagesViewController](https://github.com/jessesquires/JSQMessagesViewController) demo application to add Watson services, including Speech to Text, Conversation, and Text to Speech.
 
-_Please note that this project is still a work-in-progress!_
-
 ### Requirements
 
-- Xcode 7.3+
 - iOS 8.0+
-
-### Dependency Management
-
-This project uses both [Carthage](https://github.com/Carthage/Carthage) and [CocoaPods](https://cocoapods.org/) to manage dependencies.
-
-- Install Carthage using [Homebrew](http://brew.sh/): `brew install carthage`
-- Install CocoaPods: `sudo gem install cocoapods`
+- Xcode 9.0+
+- Swift 4.0+
 
 ### Getting Started
 
 1. Clone the repository: `git clone https://github.com/watson-developer-cloud/simple-chat-objective-c.git`
-2. Build the dependencies: `carthage update --platform iOS`
-3. Open `ChatApp.xcworkspace`
-4. Copy `CredentialsExample.swift` to `Credentials.swift`.
-5. Update your service credentials in `Credentials.swift`
-6. Build and run the app!
+2. Install [Carthage](https://github.com/Carthage/Carthage) using [Homebrew](http://brew.sh/): `brew install carthage`
+3. Build the dependencies: `carthage update --platform iOS`
```

Comment:
Do we need `3.5 Build more dependencies: pod install`?

Reply:
Looks like the `Pods` directory is checked-in here too, so it shouldn't be necessary to run a `pod install`.